news/2017-01-23-weld-300CR1.asciidoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: news
+title: Weld 3.0.0.CR1
+author: Martin Kouba
+priority: 1
+change_frequency: weekly
+excerpt: Weld 3.0.0.CR1 (CDI 2.0-PFD) released. And it's almost Final!
+desc: Weld 3.0.0.CR1 released
+tags: [release]
+---
+:linkattrs:
+
+I am very pleased to announce the release of Weld *3.0.0.CR1* (aligned with *CDI 2.0-PFD*).
+See also https://issues.jboss.org/projects/WELD/versions/12331714[the release details, window="_blank"].
+Thanks to everyone involved in this release!
+This version of Weld will be part of the JSR 365 Approval Ballot (watch http://cdi-spec.org/ for news).
```

Comment:
I would turn this into asciidoc link:
http://cdi-spec.org/ -> http://cdi-spec.org/[CDI spec]